### PR TITLE
Update dataflow-task-parallel-library.md

### PR DESCRIPTION
**Pull request by imba-tjd**
Fixes https://github.com/dotnet/docs/issues/6917

> The delegate that you provide to a TransformBlock<TInput,TOutput> object can be of type System.Func<TInput, TOutput> or type `System.Func<TInput, Task>`.

To

> The delegate that you provide to a TransformBlock<TInput,TOutput> object can be of type System.Func<TInput, TOutput> or type `System.Func<TInput, Task<TOutput>>`.

View rich diff will be more clear...

### CODE REVIEW

File: docs/standard/parallel-programming/dataflow-task-parallel-library.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ ms.author: "ronpet"
...
+ The <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> class is a target block that calls a delegate when it receives data. Think of a <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object as a delegate that runs asynchronously when data becomes available. The delegate that you provide to an <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object can be of type <xref:System.Action> or type `System.Func<TInput, Task>`. When you use an <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object with <xref:System.Action>, processing of each input element is considered completed when the delegate returns. When you use an <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object with `System.Func<TInput, Task>`, processing of each input element is considered completed only when the returned <xref:System.Threading.Tasks.Task> object is completed. By using these two mechanisms, you can use <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> for both synchronous and asynchronous processing of each input element.
```
Comment (rpetrusha) [inline docs/standard/parallel-programming/dataflow-task-parallel-library.md]: Actually, it appears, @imba-tjd, that the `System.Action` delegate is incorrect as well; it should be \<xref:System.Action %601>. There are two occurrences of it in this paragraph.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for addressing the issue and submitting this PR, @imba-tjd. There's one additional correction; ActionBlock\<T>  invokes an Action\<T> delegate, not an Action delegate.

File: docs/standard/parallel-programming/dataflow-task-parallel-library.md (hunk 1 above)
Comment (imba-tjd) [reply docs/standard/parallel-programming/dataflow-task-parallel-library.md]: Thanks, done. I also made two more changes.

**Review by rpetrusha — APPROVED** [review]
Thanks for making the additional correction, @imba-tjd. We'll merge your PR now. Your changes should be live on docs.microsoft.com in the next day or two.

**Pull request merged**